packages/styles/scss/components/content-group-horizontal/_content-group-horizontal.scss

```diff
@@ -10,23 +10,37 @@
 @import '../content-block-horizontal/content-block-horizontal';
 
 @mixin content-group-horizontal {
+  :host(#{$dds-prefix}-content-group-horizontal),
   .#{$prefix}--content-group-horizontal {
-    .#{$prefix}--content-item-horizontal__item:last-child {
-      @include carbon--breakpoint('max') {
-        padding-bottom: $layout-07;
-      }
+    /* stylelint-disable-next-line selector-type-no-unknown */
+    dds-hr,
+    .bx--content-block .bx--hr {
       @include carbon--breakpoint('lg') {
-        padding-bottom: $layout-07;
-      }
-      @include carbon--breakpoint('md') {
-        padding-bottom: $layout-06;
-      }
-      @include carbon--breakpoint('sm') {
-        padding-bottom: $layout-05;
+        margin-right: -9rem;
```

Comment:
Where is this being applied? And why would such a large value be needed on a margin?

Reply:
Talked with @shixiedesign and component is now 12 columns wide. Content (except for the HR) will take up 10 columns.